integration_testing/features/coach/coach-can-edits-group.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Edit a group’s name
+    Coach should be able to sign in to access Kolibri Coach tab
+    Coach can edit a group's name
+
+  Background:
+    Given you are in coach groups page
+    Given that there are groups created
+
+  Scenario: Edit a group name
+    When I click on OPTIONS for a group
+    Then Rename Group and Delete Group modal appears
```

Comment:
`Rename group` and `Delete group` here are not modals, but options to select. It might be picky, but can lead to confusion because on following steps we do have the `Rename group` modal. I would suggest changing this to `*Rename group* and *Delete group* options appear`.